src/main/java/frc/robot/RobotContainer.java

```diff
@@ -350,6 +394,12 @@ public Command getAutonomousCommand() {
       // An ExampleCommand will run in autonomous
       // odometry.zeroHeading();
       return chooser.getSelected();
+
+    }
+
+    public void startLEDs() {
+      ledSubsystem.startColor();
```

Comment:
Instead of making a method in robotContainer which only calls a method in a different subsystem, make a getter method in the robotContainer that returns the ledSubsystem, and use that to call `startColor` directly

Reply:
? I don't understand. I can make a getter method but them how will I use that to start color?